docs/testing.md

```diff
@@ -122,9 +122,41 @@ Hook specific assertions:
 
 ## Cypress Tests
 
-Cypress is used to run tests against the frontend while mocking all network requests.
+Cypress is used to run tests against the frontend by either mocking all network requests or directly interacting with a live cluster.
 
-Single command to run all Cypress tests or a specific test (build frontend, start HTTP server, run Cypress):
+### E2E Tests
+
+Cypress e2e tests run against a live cluster.
+
+Before running the Cypress e2e tests, you must populate the test variables by first creating a copy of the `test-variables.yml.example` file and setting the `CY_TEST_CONFIG` env variable value to be the path to this file. Update the variables according to your testing environment.
+
+```bash
+CY_TEST_CONFIG=./test-variables.yml`
```

Comment:
```suggestion
CY_TEST_CONFIG=./test-variables.yml
```
Did you mean to have the back tick at the end?

Reply:
No I did not :(